apps/omar-ui-app/grails-app/views/views/map/_map.partial.html.gsp

```diff
@@ -863,571 +847,963 @@
+                                        <li ng-click="list.sortWfs('mission_id', '+A', 'Mission');"><a>Misson <span class = "glyphicon glyphicon-arrow-up"></span></a></li>
+                                    </ul>
+                                </li>
+                                <li class="dropdown">
+                                    <a class="dropdown-toggle navbar-sort-dropdown-toggle"
+                                       data-toggle="dropdown"
+                                       role="button"
+                                       aria-haspopup="true"
+                                       aria-expanded="false">{{list.exportSelectedButtonText}}
+                                        <span class="caret"></span>
+                                    </a>
+                                    <ul class="dropdown-menu dropdown-menu-right">
+                                        <li class="dropdown-header">Use the checkboxes on the image cards to </li>
+                                        <li class="dropdown-header">select individual images. They can then be</li>
+                                        <li class="dropdown-header"> downloaded, exported, or viewed in</li>
+                                        <li class="dropdown-header">applications</li>
+                                        <li class="divider" ng-if="list.showSelectedButton"></li>
+                                        <li role="menuitem" ng-click="list.downloadSelectedImages()" ng-if="list.showSelectedButton">
+                                            <a href="">Download
+                                                <i class="fa fa-info-circle text-info" style="font-size: 12px;" aria-hidden="true" tooltip-placement="left-bottom" uib-tooltip="A maximum of 10 can be downloaded at one time"></i>
+                                            </a>
+                                        </li>
+                                        <li class="divider"></li>
+                                        <li class="dropdown-header">
+                                            Exports
+                                            <i class="fa fa-info-circle text-info" tooltip-placement="left-bottom" uib-tooltip="Export the selected images into the following formats"></i>
+                                        </li>
+                                        <li role="menuitem" ng-click="list.exportSelectedImages('CSV')">
+                                            <a href="">CSV</a>
+                                        </li>
+                                        <li role="menuitem" ng-click="list.exportSelectedImages('GML2')">
+                                            <a href="">GML2</a>
+                                        </li>
+                                        <li role="menuitem" ng-click="list.exportSelectedImages('GML3')">
+      
```

Comment:
Unless there is a specific reason all this is done in jquery, I think using angular would be preferred. ng-class, ng-show, ng-hide, etc..